types/stripe-node/index.d.ts

```diff
@@ -1,6 +1,9 @@
 // Type definitions for stripe-node 4.7.0
```

Comment:
This needs to be updated to `4.7` (not to include the PATCH version).

Reply:
Also, does anyone else think this should be renamed to just stripe now to match the NPM package name?

Reply:
Looking at the [change logs](https://github.com/stripe/stripe-node/blob/master/CHANGELOG.md) it looks like this could even be bumped up to `5.1` as the only breaking changes are its requirement for Node 4.x and over. I can't see any changes that would break these type definitions, unless anyone disagrees?

Reply:
> Also, does anyone else think this should be renamed to just stripe now to match the NPM package name?

100% yes! It was extremely confusing for me to find this package in the first place